src/fragments/lib/troubleshooting/common/upgrading.mdx

```diff
@@ -46,88 +46,77 @@ In order to use Amplify JavaScript V6, you will need to make sure you are using
-  running `npm install`
-</Callout>
+<Callout>Make sure to delete your package-lock.json and node_modules folder before running `npm install`</Callout>
 
 <BlockSwitcher>
+  <Block name="Web">
+    Upgrade to the latest Amplify library using the following command.
 
-<Block name="Web">
-
-Upgrade to the latest Amplify library using the following command.
-
-```
-npm install aws-amplify
-```
-
-The `aws-amplify` package in v6 includes the categories below:
-
-- Auth
-- API
-- Storage
-- Analytics
-- DataStore
-- In-App Messaging
-
-If you previously listed `@aws-amplify` namespaced packages for the above categories as dependencies in your `package.json`, you will need to remove them to avoid dependency duplication.
+    ```
+    npm install aws-amplify
+    ```
 
-If you would like to use Geo, Predictions, PubSub, or Interactions you will need to install those packages separately. (see [category-specific migration instructions](/[platform]/build-a-backend/troubleshooting/migrate-from-javascript-v5-to-v6/#4-upgrade-category-usage))
+    If you are using npm as a package manager, make sure the version of `aws-amplify` in your package.json file is `latest` or `^6.0.1` and re-installing dependencies by running `npm install`.
 
-</Block>
+    The `aws-amplify` package in v6 includes the categories below:
 
-<Block name="React Native">
+    - Auth
+    - API
+    - Storage
+    - Analytics
+    - DataStore
+    - In-App Messaging
 
-In v6, the following packages have been replaced with `@aws-amplify/react-native` and should be removed from your `package.json`.
+    If you previously listed `@aws-amplify` namespaced packages for the above categories as dependencies in your `package.json`, you will need to remove them to avoid dependency duplication.
 
-* `amazon-cognito-identity-js`
-* `aws-amplify-react-native`
+    If you would like to use Geo, Predictions, PubSub, or Interactions you will need to install those packages separately. (see [category-specific migration instructions](/[platform]/build-a-backend/troubleshooting/migrate-from-javascript-v5-to-v6/#4-upgrade-category-usage))
 
-Then upgrade/install the necessary dependencies using the following command:
+  </Block>
 
-```
-npm install aws-amplify @aws-amplify/react-native @react-native-community/netinfo @react-native-async-storage/async-storage react-native-get-random-values
-```
+  <Block name="React Native">
 
-Note that v6 supports react-native v0.70+, so if you prefer manually upgrading dependencies double-check the version of react-native in your package.json file.
+    In v6, the following packages have been replaced with `@aws-amplify/react-native` and should be removed from your `package.json`.
 
-`@aws-amplify/react-native` requires a minimum iOS deployment target of `13.0`, if has not already been set to `13.01 or above, open the _Podfile_ located in the _ios_ directory and update the `target` value:
+    * `amazon-cognito-identity-js`
+    * `aws-amplify-react-native`
```

Comment:
Have we provided migration instructions for legacy React Native `Authenticator` customers to migrate to using the current `Authenticator` exported from `@aws-amplify/ui-react-native`? 

Also, afaik `@aws-amplify/react-native` is basically a net new package providing polyfills and while required for usage with JS v6, does not provide any of the esports previously available from `aws-amplify-react-native` (@HuiSF Can you please confirm 😅 ).

Reply:
Good catch Caleb. `aws-amplify-react-native` exports all react-native UI components without any native modules. So it's not a replacement indeed.

However, `aws-amplify-react-native` source code is incompatible with Amplify JS v6, we may need to change the wording around removing this package while upgrading to v6.

Reply:
> Have we provided migration instructions for legacy React Native Authenticator customers to migrate to using the current Authenticator exported from @aws-amplify/ui-react-native?

Just a heads that this PR does not need to include the React Native Authenticator migration, UI team will own it @erinleigh90

> However, aws-amplify-react-native source code is incompatible with Amplify JS v6, we may need to change the wording around removing this package while upgrading to v6.

Thinking that we should explicitly call out that `aws-amplify-react-native` is not supported in v6 and remove the language regarding dependency replacement here as there is no real correlation between `aws-amplify-react-native` and `@aws-amplify/react-native`. We can follow up with the legacy RN `Authenticator` migration guide as noted above

There does still need to be migration instructions for RN customers who are not using the `Authenticator`. Again would avoid verbiage that includes "replacement" since `@aws-amplify/react-native` is not a 1:1 replacement of `amazon-cognito-identity-js`, assuming we add a callout regarding the lack of support for `aws-amplify-react-native` above this block, suggest something along the lines of:

```suggestion
To upgrade React Native projects that do not utilize `aws-amplify-react-native`, remove `amazon-cognito-identity-js` from the project _package.json_ and install `@aws-amplify/react-native`:
```

And add an install script below:

```
npm install @aws-amplify/react-native
```